pkg/cheatsheet/generate.go

```diff
@@ -33,13 +41,20 @@ func main() {
 	for lang := range i18n.GetTranslationSets() {
 		os.Setenv("LC_ALL", lang)
 		mApp, _ := app.NewApp(mConfig)
-		file, err := os.Create("./docs/keybindings/Keybindings_" + lang + ".md")
+
+		file, err := os.Create(dir + "/Keybindings_" + lang + ".md")
 		if err != nil {
 			panic(err)
 		}
 
 		bindingSections := getBindingSections(mApp)
 		content := formatSections(mApp, bindingSections)
+		content = fmt.Sprintf(
+			"_This file is auto-generated. To update, make the changes in the "+
+				"pkg/i18n directory and then run `%s` from the project root._\n\n%s",
+			generateCheatsheetCmd,
+			content,
+		)
```

Comment:
I copied over these changes from #373 as i needed the constants as well. If this gets approved/merged first we can close #373 .